recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
 build:
   number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: True  # [win or not py35]
+  skip: True  # [win]
```

Comment:
This should be `# [win or not py>=35]` since it is a recommendation. I guess py35 still works. If not you should use `# [win or not py36]` instead. Last but but least this need to be re-rendered to get the changes in. See https://github.com/conda-forge/conda-forge.github.io/pull/350/files (will become real docs soon.)